core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -549,7 +549,9 @@
   TSERV_SLOW_FILEPERMIT_MILLIS("tserver.slow.filepermit.time", "100ms", PropertyType.TIMEDURATION,
       "If a thread blocks more than this period of time waiting to get file permits,"
           + " debugging information will be written."),
-
+  TSERV_BULK_MAX_TABLET_OVERLAP("tserver.bulk.max.overlap", "0", PropertyType.COUNT,
```

Comment:
Thinking ahead, for the new bulk import code in the 2.x branch I think this check would actually execute in the master.  Therefore the prefix `tserver` is not a good prefix.  This is not a problem with this PR, more of a problem with the current property name conventions.  I searched for bulk in this file and found `master.bulk.` and `tserver.bulk.` prefixes.   The user does not care where the check is done.  Could possibly use the prefix `general` instead of `tserver`, but this creates a third prefix for bulk properties which could make this property harder to find.

Not sure there is a good solution to this problem.

Reply:
I agree, I don't think users care whether the prefix is tserver or master.  I don't think its worth going through a deprecation cycle to migrate all of the ```tserver``` properties to ```master``` or ```general```.  That just adds extra unnecessary work for users.  I think the name ```master.bulk.max.tablets``` is OK.

Reply:
>  I think the name master.bulk.max.tablets is OK.

I like that name and would be in favor of using it in 1.9 and 2.x.   The name will make more sense in 2.x than in 1.9 but that is better for the long term.